### DISCUSSION TITLE
Support nested/chained picklists

### DISCUSSION THREAD

**Discussion by maxpatiiuk** [specify/specify7#5636] [Feature Requests]
If there already is an issue for this, please merge them and close the duplicate

Reposted from Discourse: [Chained Pick Lists](https://discourse.specifysoftware.org/t/chained-pick-lists/604)

Fedor:
> Does anyone know how we could wrangle Specify form design to enable “chained” pick lists, i.e. picking something in the first list restricts the options in the next list and so on?

Andy:
> Not something that is currently possible but something that has been requested numerous times.

Nielz:
>  that sounds interesting. Do you have any examples of things for which one might want to use this?

---

Also, from the Fall 2022 Development Planning Google Doc:

Max:
> Is it okay if the "Chained pick list" uses just one database field for storing all of the chain parts. Or should there be a separate field for each chain part?

Andy:
> I think the request has been to have one picklist field based on another i.e. select something from picklist field A and have it determine or subset values in picklist field B

**Comment by grantfitzsimmons** [comment]
@mc0822 

> Describe the solution you'd like
Pick lists - Is it possible to have some filtering based on a field? (quote from the request: “‘Treatment undertaken’ : have a drop-down menu dependent on each prep type as the treatments will vary”)
> 
> Reported By
> Zsuzsanna Papp of NHMD (Danish Natural History Museums)
> 
> Additional context
> https://docs.google.com/document/d/1Yxmp64R9YWMlD__aosXa4XPn-duYe9ND/edit
> 
> Grant, Theresa, & Andy said other users have requested this as well.
